packages/app/src/utils/createAnimationFrame.ts

```diff
@@ -1,16 +1,22 @@
 import { createEffect, onCleanup, untrack } from 'solid-js'
 
-export function createAnimationFrame(fn: () => void) {
+export function createAnimationFrame(fn: () => void, interval = 10) {
```

Comment:
I would also want to render on param change, not sure if this would be easy, perhaps force re-render on top of render every n ms, ideally later, with quality setting, we can just let it render up to wanted clarity and stop afterwards.

Reply:
I would not use defaults if its not making it easier to use in another part of the codebase.

Reply:
This was just a precaution if someone else is using this, I think there is some unused code that is calling it, so in that case, but since this will be removed soon, with render settings, I guess we can leave it for now